src/server/command/debugCommand.ts

```diff
@@ -0,0 +1,109 @@
+import { AnyChannel, Client, Message } from 'discord.js';
+
+async function getCommand(
+  client: Client,
+  message: Message,
+  prefix: string,
+  command: string
+) {
+  if (message.author.bot || !message.guild) return;
+
+  const str = message.content;
+  const match = str.startsWith(command);
+  if (!match) return;
+
+  const commandArgs = str.slice(prefix.length).trim().split(' ');
```

Comment:
任意個のスペースに対応できるように, `/\s+/` による分割をおすすめします
```suggestion
  const commandArgs = str.slice(prefix.length).trim().split(/\s+/);
```